mantidproject/mantid
# Add colorbar scale setting to Workbench

**Description of work.**
This PR adds a colorbar scale setting to workbench. 

**To test:**
- Open workbench
- Open Settings -> Plots -> Scroll to the bottom and change the colorbar scale option
- Load some data and plot a colorfil. Check that the setting is applied. 
- Change around the setting and make sure everything behaves correctly.


This is part of the issue #28253, which now has one setting left to add. 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
